docs/inference-providers/providers/replicate.md

```diff
@@ -44,6 +44,16 @@ Replicate is building tools so all software engineers can use AI as if it were n
 ## Supported tasks
 
 
+### Automatic Speech Recognition
+
+Find out more about Automatic Speech Recognition [here](../tasks/automatic_speech_recognition).
+
+<InferenceSnippet
+    pipeline=automatic-speech-recognition
+    providersMapping={ {"replicate":{"modelId":"openai/whisper-large-v3","providerModelId":"openai/whisper"} } }
+/>
+
+
```

Comment:
```suggestion
```

CI fails because of replicate not supporting ASR. Already told them but in the meantime let's remove that part.